src/main/resources/messages.properties

```diff
@@ -24,6 +24,11 @@ edit.proxy.column.name=Name
 edit.proxy.save.preferences=Save Preferences
 edit.proxy.cancel=Cancel
 edit.proxy.error.message.inconsistent.field.values=All fields must be empty or must have a value.
+edit.proxy.url.instructions=App URL may include Spring EL expressions. The Spring EL context will include all properties from app-launcher.properties, all request parameters namespaced as "request" and all properties from the user-info map namespaced as "user". Examples:
+edit.proxy.url.instructions.ex1=http://${server}:${port}/service/view?param=${request.param}
+edit.proxy.url.instructions.ex2=http://${server}:${port}/service/view?userId=${user['user.login.id']}
+edit.proxy.url.instructions.ex3=http://${server}:${port}/service/view/${request['form.itemId']}
```

Comment:
Lines 27 - 30 will need to change to handle request instead becoming requestParam, but I will make that change when I test the PR.